main.py

```diff
@@ -23,7 +23,7 @@ def deploy_bot():
     _log.info("....:Beginning dAppCon Bot Service:....")
 
     try:
-        key = sys.argv[1]
+        key = "907839309:AAFguq_YSfKmiApaCy9BxSPBA1EnDkjLY7s"
```

Comment:
We should revert this line to what it used to be and let dev-ops use their API key...
```suggestion
        key = sys.argv[1]
```